examples/tutorial_builder/README.md

```diff
@@ -87,7 +89,7 @@ MyApp 1.0
 ```
 
 You can use `command!()` to fill these fields in from your `Cargo.toml`
-file.  **This requires the `cargo` feature flag.**
+file.  This requires the `cargo` feature flag.
```

Comment:
What was the motivation for removing the bolding?

Reply:
I thought it made it less legible. The ex. 2 sentence is gone in my latest update.

Reply:
Pedagogically, this is when we introduce `command`.  We should be noting the feature flag that it requires.

As this is unrelated to the core of your change, let's just revert this part completely.